core/src/banking_stage/consume_worker.rs

```diff
@@ -61,8 +61,18 @@ impl<Tx: TransactionWithMeta> ConsumeWorker<Tx> {
 
     pub fn run(self) -> Result<(), ConsumeWorkerError<Tx>> {
         loop {
-            let work = self.consume_receiver.recv()?;
-            self.consume_loop(work)?;
+            match self.consume_receiver.try_recv() {
+                Ok(work) => {
+                    self.consume_loop(work)?;
+                }
+                Err(TryRecvError::Empty) => {
+                    const SLEEP_DURATION: Duration = Duration::from_millis(5);
```

Comment:
not sure what the right number is, but my gut feeling is that 5ms is too much? I'd conservatively start with 1ms, we don't really care about burning a little CPU here when not busy, these are some of the highest prio threads

Reply:
changed to 1ms